config/remote/remote.go

```diff
@@ -182,12 +192,15 @@ func (c *CdnRemoteConfig) load() {
 	c.mu.Lock()
 	defer c.mu.Unlock()
 
-	for _, f := range c.features {
-		if err := f.load(c.localCachePath, jsonFileReaderWriter{}, jsonValidator{}); err != nil {
-			log.Println(internal.ErrorPrefix, "failed loading feature [", f.name, "] config from the disk:", err)
+	for _, f := range c.features.keys() {
+		feature := c.features.get(f)
+		if err := feature.load(c.localCachePath, jsonFileReaderWriter{}, jsonValidator{}); err != nil {
+			log.Println(internal.ErrorPrefix, "failed loading feature [", feature.name, "] config from the disk:", err)
+			c.analytics.NotifyLocalUse("cli", feature.name, err)
 			continue
 		}
-		log.Println(internal.InfoPrefix, "feature [", f.name, "] config loaded from:", c.localCachePath)
+		log.Println(internal.InfoPrefix, "feature [", feature.name, "] config loaded from:", c.localCachePath)
+		c.analytics.NotifyLocalUse("cli", feature.name, nil)
```

Comment:
Maybe this `"cli"` should be a `const`? Or maybe even we could have an enum for clients